### ISSUE TITLE
Unable to use images from camera

### ISSUE THREAD

**Issue by andrea-nisco** [ArthurHub/Android-Image-Cropper#892]
I have read that I should use [https://github.com/CanHub/Android-Image-Cropper](url) but it is only available in Kotlin and I need it in Java. 
I have also read that there is actually a sample of this in Java thanks to #875, but in reality, there is no trace of this anywhere.  So is it possible to use this library in Java or not? If true, is there a real working link ?

**Comment by chenlize96** [comment]
@andrea-nisco Did you figure out it? I have the same problem.

**Comment by andrea-nisco** [comment]
> @andrea-nisco Did you figure out it? I have the same problem.

Yes, the issue is impossible to solve, but you can take te picture normally, save the uri of the image and crop it programmatically. That's how I solved it.

**Comment by chenlize96** [comment]
> > @andrea-nisco Did you figure out it? I have the same problem.
> 
> Yes, the issue is impossible to solve, but you can take te picture normally, save the uri of the image and crop it programmatically. That's how I solved it.

I succeeded in solving this issue. Here is the reference link:
https://stackoverflow.com/questions/74814279/how-to-use-canhub-image-cropper-latest-version-on-java
